AnyText/README.md

```diff
@@ -0,0 +1,92 @@
+occurrences in a document.
+* **Rename:** Enables the renaming of a symbol in a document (see find references).
+* **Document Symbols/Outline:** Enables the symbols of the document to be shown in the outline 
+(VSCode default left-hand side and top of document). The outline at the top shows the current 
+position in the document and both outlines can be used to jump between symbols in the document.
+
+### Feature configuration
+
+While the features were implemented generically, some of them can be configured or require 
+manual configuration to function properly. Manual configurations can be added to the rules 
+in the **\<GrammarName\>.manual.cs** file located under */AnyText/Grammars*.
+
+**Folding:** 
+Folding can be configured in multiple ways. A folding range can be a comment, a number of 
+imports/includes statements, a region or a generic range. While comments don't need to be 
+configured manually, the default implementations of the remaining kinds of folding ranges 
+can and often should be overridden. Overrides need to be done for each rule individually.
+
+* **public override bool IsRegionStartLiteral(string literal)**: 
+True, if the given literal denotes the start of a foldable region.
+* **public override bool IsRangeStartLiteral(string literal)**: 
+True, if the given literal denotes the start of a generic foldable region.
+* **public override bool IsMatchingEndLiteral(string literal, string startLiteral)**: 
+True, if the given literal is a matching end literal for a given start literal.
+
+Alternatively, it is also possible to directly mark specific rules as foldable ranges.
+
+* **public override bool IsImports()** : Can be set to true to denote a foldable region.
+* **public override bool IsFoldable()**: Can be set to true to denote a generic foldable range.
+
+**Selection:** No manual configuration required.
+
+**GoTo Definition:** 
+Definitions can be configured by defining which rules are considered definitions. 
+While the default implementation for definitions should generally suffice, it is important
+to define which rules denote identifiers. See Find References feature.
+
+* **public override bool IsIdentifier()**: Can be set to true to denote an identifier.
+
+If necessary, the default implementation for definitions can also be overridden.
+
+* **public override bool IsDefinition()**: Can be set to true to denote a definition.
+
+**Find References:** 
+References can be configured by defining which rules are considered references. 
+While the default implementation for references should generally suffice, it is important
+to define which rules denote identifiers. See GoTo Definition feature.
+
+* **public override bool IsIdentifier()**: Can be set to true to denote an identifier.
+
+If necessary, the default implementation for references can also be overridden.
+
+* **public override bool IsReference()**: Can be set to true to denote a reference.
```

Comment:
Das sollte man eigentlich manuell überschreiben, sonst könnte Unfug dabei rauskommen

Reply:
Also wenn es im Metamodell vermerkt ist, was der Identifier ist, dann generiert der Codegenerator automatisch eine überschriebene Property IsIdentifier, dann *kann* man es gar nicht mehr selbst überschreiben, weil es in derselben Klasse ja schon überschrieben ist. Wenn es nicht im Metamodell steht, muss man es derzeit selbst machen.

Es schadet aber trotzdem nichts, die beiden Properties zu erklären, nur vielleicht mit dem Hinweis versehen, dass sich da eigentlich Framework und Generator drum kümmern sollten